patches/server/0296-Async-command-map-building.patch

```diff
@@ -9,14 +9,15 @@ commands if the server is restarting. Using the default async pool caused issues
 due to the shutdown logic generally being much later.
 
 diff --git a/src/main/java/net/minecraft/commands/Commands.java b/src/main/java/net/minecraft/commands/Commands.java
-index 2bf67468a6c745bc6243c65210477ba129bfcb07..91b32a4856a4c9d6dc12871ed080f7a67311ead0 100644
+index 2bf67468a6c745bc6243c65210477ba129bfcb07..5e390e2e5e3af66cd7b596ecde62c8a29a00ffe6 100644
 --- a/src/main/java/net/minecraft/commands/Commands.java
 +++ b/src/main/java/net/minecraft/commands/Commands.java
-@@ -360,6 +360,23 @@ public class Commands {
+@@ -360,6 +360,25 @@ public class Commands {
          if ( org.spigotmc.SpigotConfig.tabComplete < 0 ) return; // Spigot
          // CraftBukkit start
          // Register Vanilla commands into builtRoot as before
 +        // Paper start - Async command map building
++        // Don't send to pool if it's shutdown
```

Comment:
I think this comment was left over from before you found out about the discard policies?

Reply:
Yeah see paper contrib, I will change this later.

Reply:
done